plugin-server/src/main/graphile-worker/graphile-worker.ts

```diff
@@ -55,7 +66,14 @@ export class GraphileWorker {
         retryOnFailure = false
     ): Promise<void> {
         const jobType = 'type' in job ? job.type : 'buffer'
-        const jobPayload = 'payload' in job ? job.payload : job.eventPayload
+
+        let jobPayload: Record<string, any> = {}
+        if ('payload' in job) {
+            jobPayload = job.payload
+        } else if ('eventPayload' in job) {
+            jobPayload = job.eventPayload
+        }
+
```

Comment:
just "made this work" with the changes I made - looks like a code smell though, something for a follow-up refactor